samples/excel/16-custom-functions/volatile-function.yaml

```diff
@@ -7,18 +7,14 @@ api_set:
     CustomFunctionsRuntime: 1.1
 script:
     content: |
-        /** @CustomFunction */
-        function ROLL6SIDEDICE(x: number): number {
-          let i:number;
-          let result:number;
-          result = 0;
-          for (i = 0;i<x;i++) {
-            var roll:number;
-            roll = Math.random()*6;
-            roll = Math.round(roll);
-            result += roll;
-          }
-          return result;
+        /** @CustomFunction 
+        *  @description Simulates rolling a 6 sided die.
+        * @returns A whole number from 1 to 6.
+        * @volatile
```

Comment:
So does `@volatile` work now?

Reply:
@Zlatkovsky I'm not sure. I use manual recalc to check that the functions are called again (which I understand is the volatile behavior) but they always work even if I leave volatile off. However, I notice script lab adds the volatile keyword to the metadata if I use it so I put it in.